content/home/hero.md

```diff
@@ -47,4 +47,4 @@ hero_media = "2i2c-hub-overview.png"
 #  label = '2i2c provides open-source infrastructure for interactive computing at institutions of research and education.'
 +++
 
-2i2c is a non-profit initiative that develops, deploys, customizes, and manages open source tools for interactive computing in research and education.
+2i2c is a non-profit initiative that runs customized JupyterHubs for educational & research users.
```

Comment:
The main reason I had all those verbs in there was to make it clear that we aren't _only_ a "managed service provider", but also do development, support communities, write our own grants / collaborate on grants, etc. What do you think about this? I like the more streamlined version you've got here, but worry that we under-sell the stuff 2i2c does

Reply:
Let's structure that information below the lede? IMO putting so many verbs in the lede makes it ineffective, and we can add more information underneath.

Reply:
Yeah I agree with that take...hmm, what if we just modified it slightly to

```
2i2c is a non-profit organization that manages customized JupyterHubs for educational & research users
```

I believe we swapped to using "organization" rather than initiative in the latest iteration on the "governance / structure" doc

Reply:
@choldgraf how about now?

Reply:
manages and supports seemed too similar, so I instead used runs and supports.

Reply:
cool 👍 . I think we may change this if/when we start doing more stuff, but for now it is more accurate this way. Does that make sense to you?

Reply:
@choldgraf yep, exactly! Makes sense to me :)